docs/azure-saas-docs/content/en/quick-start.md

```diff
@@ -0,0 +1,76 @@
+---
+type: docs
+title: "Quick Start"
+linkTitle: "Quick Start"
+weight: 10
+description: "Getting Started with the Azure SaaS Dev Kit"
+---
+
+On this page, you will find instructions to run the dev kit in your local environment, how to deploy the solution to Azure, and where to put your application code to customize the solution.
+
+> Tip: If you're new here and want to learn what this is, check out the [welcome page](..)
+
+## 1. Setup Identity Framework
+
+This project uses [Azure Active Directory B2C](https://docs.microsoft.com/en-us/azure/active-directory-b2c/overview) for an IDP (Identity Provider). The first step in setting up this project is to configure a new B2C instance to house your local user accounts. You will also need to deploy the [Permissions API](../components/identity/permissions-service), as B2C will have a dependancy on it.
+
+To setup the Identity Framework, we have provided a powershell script [here]() that automates the setup for you. This powershell script will output a parameters file that you'll need to provide when deploying the solution to azure in step 2.b.
+
+After finishing the IDP setup, you may choose to either run the project locally first or immediately deploy the solution to Azure.
+## 2.a. Running the Dev Kit in your local dev environment
+
+- Install the latest version of [Visual Studio 2022](https://visualstudio.microsoft.com/vs/). You may also use Visual Studio Code, but the solution and projects are targetted at VS2022.
+- Clone the repository `https://github.com/Azure/azure-saas.git` on to your dev machine.
+- Open the `.sln` in the root of the repository. This solution includes all of the modules.
+- Depending on the project you wish to run, you'll need to set some secrets to properly setup authentication with B2C. See the [App Settings](#app-settings) section below
+
+### App Settings
+When running the project locally, you will need to set some app settings & secrets manually using the [.NET secrets manager](https://docs.microsoft.com/en-us/aspnet/core/security/key-vault-configuration?view=aspnetcore-6.0#secret-storage-in-the-development-environment). When deployed to azure (below), these secrets are automatically configured for you and stored in Azure Key Vault.
```

Comment:
Since this section talks about both local and deployed scenarios, the overall App Settings section should be moved on its own. To add additional clarity in the formatting, perhaps bullet the app setting scenarios as:
- Running locally you will need to set some App Settings & User Secrets manually using the [.NET Secret Manager](https://docs.microsoft.com/aspnet/core/security/key-vault-configuration?view=aspnetcore-6.0#secret-storage-in-the-development-environment).
- Deployed to Azure, these secrets are automatically configured for you and stored in the [Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/).